src/app/external-log-in/registration-types/orcid-confirmation/orcid-confirmation.component.html

```diff
@@ -0,0 +1,35 @@
+<form class="form-login"
+      [formGroup]="form" novalidate>
+  <label class="font-weight-bold mb-0 text-uppercase">{{ registrationData.registrationType }}</label>
+  <input [attr.aria-label]="'netId' | translate"
```

Comment:
Why are we passing a literal string of `netId` to translate?  Shouldn't this be an i18n key?

This field also looks to be missing a `label`?  We might consider just adding a corresponding `<label>` instead of the `aria-label`.